src/main/cliapp/src/components/GenericDataTable/FilterComponent.js

```diff
@@ -39,6 +43,8 @@ export const FilterComponent = ({
 				<FilterComponentDropDown
 					isEnabled={isEnabled}
 					field={fields[0]}
+					nonNullFields={nonNullFields}
+					nullFields={nullFields}
```

Comment:
@cmpich lets take these out as they are not implemented in the FilterComponentDropDown or FilterMultiSelectComponent

Reply:
Ok. Also removing from FilterMultiSelectComponent as well.

Reply:
I pushed in the code already.